docs/variables/add-variable.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Add a Variable
+intro: Adding variables ensures a more consistent resource deployment by storing crucial information and settings for each resource environment.
```

Comment:
```suggestion
intro: Variables are key/value pairs accessible by Pipeline Step commands. Use environment Variables to store configuration data used by commands in your Pipeline steps (such as compiler flags, environment specific configuration, credentials and secrets). Note: If a resource requires multiple Variables to be written to a custom file (e.g.: an Application `.env` file), we recommend storing them using "Configuration Files" instead of multiple variables.
```